ci/scripts/test-concourse-runner.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/ash
```

Comment:
This should be `#!/bin/bash`

The docker-helpers script it sources is explicitly bash, I'm unsure if it's actually ash capable or not, but it's better to be on the safe side and keep the shell the same.